### PR TITLE
docs: Add the changelog of 4.0.0-rc.4

### PR DESCRIPTION
**Pull request by dengfuping**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | `None`          |
| 🇨🇳 Chinese |  `None`         |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/changelog-4.0.0-rc.4/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/changelog-4.0.0-rc.4/CHANGELOG.zh-CN.md)
[View rendered docs/spec/dark.md](https://github.com/ant-design/ant-design/blob/changelog-4.0.0-rc.4/docs/spec/dark.md)

### CODE REVIEW

**Comment by yoyo837** [comment]
#21298

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,42 @@ timeline: true
 
 ---
 
+## 4.0.0-rc.4
+
+`2020-02-09`
+
+- 📖 Add [color palette](https://preview-21101-ant-design.surge.sh/docs/spec/dark-cn#%E5%9F%BA%E7%A1%80%E8%89%B2%E6%9D%BF) and [palette generation tool](https://preview-21101-ant-design.surge.sh/docs/spec/dark-cn#%E8%89%B2%E6%9D%BF%E7%94%9F%E6%88%90%E5%B7%A5%E5%85%B7) for dark theme. [#21101](https://github.com/ant-design/ant-design/pull/21101)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: This is a document update. No need to list in changelog.

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,42 @@ timeline: true
...
+- 🌟 Add `bordered` prop for Select、TreeSelect、DatePicker、TimePicker and Cascader. [#21242](https://github.com/ant-design/ant-design/pull/21242)
+- 🌟 Add `selectAllLabels` prop for Transfer. [#21139](https://github.com/ant-design/ant-design/pull/21139) [@morenyang](https://github.com/morenyang)
+- 💄 Redesign the style of ink bar for Tabs. [#21256](https://github.com/ant-design/ant-design/pull/21256)
+- 💄 Add less variable `@form-item-label-font-size`. [#21216](https://github.com/ant-design/ant-design/pull/21216)
+- 🐞 Fix Badge style for Badge with Typography. [#21235](https://github.com/ant-design/ant-design/pull/21235)
+- 🐞 Fix Checkbox not work when Checkbox and Checkbox.Group are separated by other component. [#21146](https://github.com/ant-design/ant-design/pull/21146) [@morenyang](https://github.com/morenyang)
+- 🐞 Fix Collapse.Panel wrong content width when `extra` prop is set. [#21202](https://github.com/ant-design/ant-design/pull/21202) [@zhiyuc123](https://github.com/zhiyuc123)
+- 🐞 Fix ConfigProvider `autoInsertSpaceInButton` prop not work. [#21176](https://github.com/ant-design/ant-design/pull/21176)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: > Refactor Button disable autoInsertSpaceInButton will not calculate text length.

**Comment by yoyo837** [comment]
#21290  没有加

File: CHANGELOG.en-US.md (hunk 2 above)
Comment (dengfuping) [reply CHANGELOG.en-US.md]: - 虽然修改的是 Button 的实现，但用户的设置入口还是在 ConfigProvider。所以建议还是保留现在的写法，用户的感知更强一点。

File: CHANGELOG.en-US.md (hunk 1 above)
Comment (dengfuping) [reply CHANGELOG.en-US.md]: - 暗色模式下的色板和色板生成工具，是站点提供的新功能，在 changelog 里提下比较好。

File: CHANGELOG.en-US.md (hunk 2 above)
Comment (dengfuping) [reply CHANGELOG.en-US.md]: - 那 changelog 里就不提了，以去掉。

**Comment by dengfuping** [comment]
> #21290 没有加

- 已加。

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,43 @@ timeline: true
...
+- 🐞 修复 Steps 在 size="small" 和 labelPlacement="vertical" 时图标没有对齐的问题。[#21258](https://github.com/ant-design/ant-design/pull/21258)
+- Table
+  - 🐞 修复数据项没有 `children` 字段时 `expandIcon` 属性不生效的问题。[#21169](https://github.com/ant-design/ant-design/pull/21169)
+  - 🐞 修复 Column 的 `sorter` 属性不生效的问题。[#21194](https://github.com/ant-design/ant-design/pull/21194)
+  - 🐞 修复自定义筛选无法输入的问题。[#21218](https://github.com/ant-design/ant-design/pull/21218)
+- 🐞 修复 TimePicker 的 `defaultOpenValue` 属性不生效的问题。[#21198](https://github.com/ant-design/ant-design/pull/21198)
+- Transfer
+  - 🐞 修复头部选择框文案的单位展示不正确的问题。[#21136](https://github.com/ant-design/ant-design/pull/21136) [@morenyang](https://github.com/morenyang)
+  - 🐞 修复搜索框中的搜索图标没有垂直居中的问题。[#21247](https://github.com/ant-design/ant-design/pull/21247)
+- 🐞 修复 Typography 在可编辑状态时光标没有在输入框末位的问题。[#21268](https://github.com/ant-design/ant-design/pull/21268)
```
Comment (yoyo837) [inline CHANGELOG.zh-CN.md]: 末位？末尾？

Comment (yoyo837) [reply CHANGELOG.zh-CN.md]: `位`好像也行。

Comment (dengfuping) [reply CHANGELOG.zh-CN.md]: - 已改成 `末尾`，更容易理解。

**Review by yoyo837 — APPROVED** [review]

**Pull request merged**